core/js/typespecs/app_types.ts

```diff
@@ -554,6 +554,7 @@ export const projectScoreSchema = z
         destination: z.literal("expected").nullish(),
       })
       .nullish(),
+    position: z.string().nullish(),
```

Comment:
Since these are public-facing types (it'll end up in the REST API), we should add some docs about it with `.describe`.

If we really want to hide this from the user, we could also augment the `projectScore` type signature with this column internally wherever we need it, but I don't think it's super necessary to hide.

Reply:
Great, will add a description via `.describe` 👍